extractor/main.go

```diff
@@ -146,6 +159,7 @@ func execExtract(ctx context.Context, cfg *extractorCfg) error {
 				if writeErr := writePackageMetadata(metadataFromMsg(msg), outputDir); writeErr != nil {
 					return writeErr
 				}
+
```

Comment:
Leftover newline?

Reply:
Removed it [e33230b](https://github.com/gnolang/tx-exports/pull/11/commits/e33230b782b6b1ea002ed8c924a4243513b5fd93)